app/src/main/java/com/belatrixsf/allstars/adapters/EmployeeListAdapter.java

```diff
@@ -23,6 +23,10 @@
  */
 public class EmployeeListAdapter extends RecyclerView.Adapter<EmployeeListAdapter.EmployeeViewHolder> {
 
+    static final String EMPTY_STRING = "";
+    static final String NUMERIC_SYMBOL = "#";
+    static final int CAP_POSITION = 0;
```

Comment:
I think this constants can be used in different places in the app. Maybe we can create a #Globals or #Constants  and put them there. @PedroCarrillo opinions?